contracts/interfaces/IAsset.sol

```diff
@@ -45,16 +45,27 @@ interface IAsset is IRewardable {
 
     /// @param {UoA} The max trade volume, in UoA
     function maxTradeVolume() external view returns (uint192);
-}
 
-interface TestIAsset is IAsset {
```

Comment:
What's the reason for getting rid of the TestIAsset interface? imo the `IAsset` interface should be the interface that the _protocol_ requires of an asset. This is a lot smaller than what we currently have. For example, not every asset is going to need or have:
- `chainlinkFeed()`
- `oracleTimeout()`
- `tryPrice()`

An asset that is priced using TWAPs does not have a concept of an oracleTimeout, because it always has a price.

Reply:
Is there a cost to the `TestIAsset` interface I'm not seeing?

Reply:
yeah i guess i didn't think of the TWAP case since we don't have any of those yet. we can revert on this if you think it makes more sense to keep TestIAsset

Reply:
I do think we should do that.